graphdatascience/graph_data_science.py

```diff
@@ -57,6 +74,21 @@ def __init__(self, endpoint: Union[str, QueryRunner], auth: Any = None, aura_ds:
             raise InvalidServerVersionError(f"{server_version_string} is not a valid GDS library version")
         self._server_version = ServerVersion(*map(int, server_version_match.groups()))
 
+        if arrow and self._server_version >= ServerVersion(2, 1, 0):
+            try:
+                arrow_info: Series = self._query_runner.run_query("CALL gds.debug.arrow()").squeeze()
+                if arrow_info["running"]:
+                    disable_server_verification = driver._pool.pool_config.trust == "TRUST_ALL_CERTIFICATES"
```

Comment:
As @robsdedude warned us on slack we  cannot really rely on these internals and need to find another way of extracting this info

Reply:
We will always set this to true for now and hopefully there will be a way to poll in the driver API in the future